ruma-events/src/room/aliases.rs

```diff
@@ -27,6 +27,18 @@ impl AliasesEventContent {
     pub fn new(aliases: Vec<RoomAliasId>) -> Self {
         Self { aliases }
     }
+
+    /// Redact an `AliasesEventContent` according to current Matrix spec.
+    pub fn redact(self) -> RedactedAliasesEventContent {
+        RedactedAliasesEventContent { aliases: None }
+    }
+
+    /// Redact an `AliasesEventContent` according to version 1 of the Matrix spec.
+    ///
+    /// This leaves the `aliases` information intact after redaction.
+    pub fn redact_v1(self) -> RedactedAliasesEventContent {
```

Comment:
If we do it like this it's always going to be a breaking change when the redaction rules change again (bc. the old redaction method changes its name). Let's discuss the ways around that on Matrix.